probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/RunConfigurations.scala

```diff
@@ -27,53 +30,79 @@ import scala.collection.convert.ImplicitConversions.`collection AsScalaIterable`
 import scala.concurrent.ExecutionContext
 
 object RunConfigurations extends IntelliJApi {
```

Comment:
Does this object also has runner for apps? If so it is ok, if not, maybe rename it for TestRunConfigurations? Anyway, current method names are quite confusing. Both named execute with very different behavior. Parameter type doesn't explain anymore what it is about. We should have something like `executeJUnit` and `executeGenerated`. Also I believe we don't need to have the `TestRunConfigurationMatch` case class, at least not here, I'd unpack the arguments.
I think such api would be much nicer:
```
object TestRunners {
  def runJUnit(scope: TestScope): TestRunResult = ???
  def runGenerated(scope: TestScope, runnerToSelect: Option[String]): TestRunResult = ???
}
```

Reply:
Actually, it has `def execute(runConfiguration: ApplicationRunConfiguration)`

Reply:
ok, so see my edit